hca_ingest/importer/importer.py

```diff
@@ -82,8 +83,18 @@ def import_file(self, file_path, submission_url, is_update=False, project_uuid=N
                 submission = self._submit_new_entities(entity_map, submission_url)
 
             project = entity_map.get_project()
+
+            def is_readable(entity):
+                """check that the entity is readable"""
+                return entity.ingest_json.editable == 'true'
```

Comment:
The comment is unnecessary; a meaningful function name is enough.
In this case the current signature is good.